dll/win32/kernel32/wine/actctx.c

```diff
@@ -10,16 +10,14 @@
  *                  Samuel Serapi�n
  */
 
-/* Partly synched with Wine 1.7.17 */
+/* Synched with Wine 1.7.55 */
```

Comment:
```suggestion
/* Synched with Wine 3.3 */
```
Or simply remove this line?

Reply:
no

Reply:
May I ask why this does not match WINESYNC.txt?